web-backend/src/main/java/com/statusneo/vms/config/AppConfig.java

```diff
@@ -0,0 +1,14 @@
+package com.statusneo.vms.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.client.RestTemplate;
+
+@Configuration
+public class AppConfig {
+
+    @Bean
```

Comment:
Don't use RestTemplate, use the newer RestClient. I replaced the production code to use RestClient